openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts

```diff
@@ -1444,7 +1444,7 @@ API:
 -   Bulk Import API now creates entities if they are not present during the import.
 -   Table's TestSuite is migrated to EntityReference. Previously it used to store entire payload of TestSuite.
 `,
-      [`Automator ${CollateIconWithLinkMD}`]: `-  Easily maintain high-quality metadata at scale with automations. The Automator streamlines governance processes from ownership assignments to tagging, ensuring compliance and consistency.
+      [`Automations ${CollateIconWithLinkMD}`]: `-  Easily maintain high-quality metadata at scale with automations. The Automations streamline governance processes from ownership assignments to tagging, ensuring compliance and consistency.
```

Comment:
@ShilpaVernekar can you review this once? Since we change the name

Reply:
@chirag-madlani - I'm not sure what would be the final name for that feature.

Reply:
We are gonna call it `Automations` from `Automator`